src/AsyncGenerator.yml

```diff
@@ -156,8 +156,6 @@
   applyChanges: true
   analyzation:
     methodConversion:
-    - conversion: Ignore
-      hasAttributeName: IgnoreAttribute
```

Comment:
This rule was causing tests from a base class overridden for marking them as NUnit `[Ignored]` to be have their override ignored by async generator, thus having their async counterpart NUnit executed instead of being ignored.

Example of such a case in this [fixture](https://github.com/fredericDelaporte/nhibernate-core/blob/master/src/NHibernate.Test/Immutable/EntityWithMutableCollection/Inverse/EntityWithInverseOneToManyJoinTest.cs#L20).

Reply:
Ok